conferences/README.md

```diff
@@ -17,3 +17,5 @@ THANK YOU to every member of the community who spreads the word! :sparkling_hear
 | Talk at MRC-BSU | 20 March 2019 | | [Whitaker_MRCBSU.pdf](presentations/MRCBSU_20190320/Whitaker_MRCBSU_TheTuringWay_March2019.pdf)<br>(File name shortened for better readability) | [10.5281/zenodo.2599904](https://doi.org/10.5281/zenodo.2599904)
 | [Tools, Practices and Systems Scoping Workshop](https://www.turing.ac.uk/events/tools-practices-and-systems-data-science-and-artificial-intelligence-scoping-workshop) | 21 March 2019 | | [TuringWayPoster_TPSScopingWorkshop.pdf](presentations/TPS_ScopingWorkshop_20190321/Whitaker_TuringWayPoster_TPSScopingWorkshop.pdf) | [10.5281/zenodo.2598546](https://doi.org/10.5281/zenodo.2598546) |
 | [RSE Con 19](https://rse.ac.uk/conf2019/) | 17-19 September 2019 | [RSECon19-BinderHub-workshop-application.md](abstracts/RSECon19-BinderHub-workshop-application.md) | | |
+| [csv,conf,v4](https://csvconf.com) | 8 & 9 March 2019 | | [Whitaker_CSVconf_May2019.pdf](presentations/csvconf_20190509/Whitaker_CSVconf_May2019.pdf) | [10.5281/zenodo.2669548](https://doi.org/10.5281/zenodo.2669548) |
+| [Software Citation Workshop](https://www.eventbrite.co.uk/e/software-citation-workshop-tickets-59519083180) | 13 & 14 May 2019 | | [Whitaker_CSVconf_May2019.pdf](presentations/SoftwareCitationWorkshop_20190513/Whitaker_SoftwareCitation_May2019.pdf) | [10.5281/zenodo.2783998](https://doi.org/10.5281/zenodo.2783998) |
```

Comment:
Rename the slides from the Software Citation workshop to Whitaker_SoftwareCitation_May2019.pdf (vs Whitaker_CSVconf_May2019.pdf copy & pasted from previous row)?